### PR TITLE
Feat Delete Question

### PR DESCRIPTION
**Pull request by Hidayathamir**

### CODE REVIEW

**Comment by Hidayathamir** [comment]
https://github.com/Hidayathamir/opendiscuss_nextjs/assets/57469556/3b0adbb7-9165-4a53-9967-e9fc4ed84a82

**Comment by Hidayathamir** [comment]
kindly approve n merge sir @addhory

File: src/component/QuestionHighlight.tsx [hunk 1]
```diff
@@ -18,73 +20,148 @@ export default function QuestionHighlight({
...
+  }
+
+  const _handleDeleteQuestion = (questionId: number) => {
+    setModalBody("Are you sure want to delete this question?")
+    setOnModalConfirm(() => {
+      _deleteQuestion(questionId)
+      setIsModalShow(false)
+      return
+    })
+    setIsModalShow(true)
```
Comment (addhory) [inline src/component/QuestionHighlight.tsx:84]: yat i think `setIsModalShow(true)` need to delete ya since it would not be executed

Comment (addhory) [reply src/component/QuestionHighlight.tsx:84]: im lost here what happen to this code

**Review by addhory — APPROVED** [review]

**Pull request merged**